webpackShims/ui-bootstrap.js

```diff
@@ -1,11 +1,27 @@
 define(function (require) {
   require('angular');
   require('ui/angular-bootstrap/index');
+  const chrome = require('../src/ui/public/chrome/chrome');
 
   return require('ui/modules')
-  .get('kibana', ['ui.bootstrap'])
+  .get('kibana', ['ui.bootstrap', 'pascalprecht.translate'])
   .config(function ($tooltipProvider) {
     $tooltipProvider.setTriggers({ 'mouseenter': 'mouseleave click' });
-  });
+  })
+  .factory('translationsLoader', function ($q) {
+      return function (options) {
+        var deferred = $q.defer();
+        const translations = chrome.getTranslations();
+        deferred.resolve(translations);
+        return deferred.promise;
+      };
+  })
+  .config(['$translateProvider', function ($translateProvider) {
+    $translateProvider.preferredLanguage('en');
```

Comment:
IIRC the language is not variable at this point, so perhaps we could simplify this (and remove the `translationLoader` factory) by just using `$translateProvider.translations('default', chrome.getTranslations())`